### ISSUE TITLE
invalid_resource_attribute_value_type false alarm for number attribute values when sending check results to Soda Cloud

### ISSUE THREAD

**Issue by pwrignall** [sodadata/soda-core#1760]
Hi, I think this is a bug - if not, please close it!
Soda Core 3.0.18

When adding a numeric attribute e.g.
```
checks for some_data_source:
  - row_count > 0:
      attributes:
        another_check_attribute: 7
```
the check validates OK locally but when sending the results to Soda Cloud, I see this:
```
ERROR  | Error while executing Soda Cloud command response code: 400
DEBUG  | {
  "code": "invalid_resource_attribute_value_type",
  "message": "invalid resource attribute value [7.0] for resource attribute NumberResourceAttribute(super\u003dResourceAttribute(id\u003dfddd9896-ab0c-4d0d-9e4b-da076d613ab0, organisationId\u003db10aa5c3-35d4-4608-b610-1bd9b1a4e3e8, label\u003danother check attribute, name\u003danother_check_attribute, description\u003d, resourceTypes\u003d[MONITOR], created\u003d2023-01-11T11:36:46, lastModified\u003d2023-01-11T11:36:46))"
}
```
(Left UUIDs in here in case they help.) Please let me know if you need further details.

**Comment by jmarien** [comment]
SODA-1366

**Comment by vijaykiran** [comment]
Thank you @pwrignall  - I'm checking with Cloud team

**Comment by dirkgroenen** [comment]
Thanks @pwrignall - Cloud team here - would you mind sending us the Cloud Trace ID you receive at the of your scan so we can take a closer look at the issue?

**Comment by pwrignall** [comment]
Thanks, it's:
`7690889960372506552`

**Comment by pwrignall** [comment]
As far as I can see, this has been resolved as I don't see the same error in 3.0.22.
